Source/nthread.cpp

```diff
@@ -85,6 +85,9 @@ BOOL nthread_recv_turns(BOOL *pfSendAsync)
 		last_tick += 50;
 		return TRUE;
 	}
+#if defined(__3DS__)
+	return 0;

```

Comment:
```suggestion
	return FALSE;
```

Is this also needed with NONET enabled? Do you know the underlying reason for having to remove this section of code?

Reply:
It's a workaround for an undefined reference to 'SNetReceiveTurns' during compilation.
With NONET enabled.